### PR TITLE
Add support to specify files

### PR DESCRIPTION
**Pull request by felipemfp**
First of all, thanks for this amazing package! :tada: :clap:

```
mutmut --runner="python manage.py test" --tests-dir="core/tests" core/services.py,core/forms.py
```

### CODE REVIEW

File: mutmut/__main__.py [hunk 1]
```diff
@@ -190,16 +190,23 @@ def exclude(context):
...
+                exclude=exclude,
+                dict_synonyms=dict_synonyms,
+            )
+        )
+
     for path in paths_to_mutate:
+        if isfile(path) and path.endswith('.py'):
+                filename = path
+                add_mutations_by_file(mutations_by_file, filename, exclude, dict_synonyms)
+                continue
```
Comment (boxed) [inline mutmut/__main__.py]: I'd prefer an else block here, I think that's a lot clearer than a continue and fall through.

**Comment by boxed** [comment]
Cool. This has been on my mental todo for a while, just haven't gotten around to  it. I have a minor quibble (see above) but then please add yourself as a contributor and I'll merge this.

**Comment by felipemfp** [comment]
> then please add yourself as a contributor

Sorry, where should I do that?

**Comment by boxed** [comment]
Oh, I didn’t have an authors file heh. Well ok I’ll fix that. Should I use the name and email on your github profile?

**Pull request merged**